Name: SystemService.js
Version: 2.1.0-1
Author: The Resetter
Description: Add 3, 8 and 10 Digits to Dialing Shortcuts

--- .orig/usr/palm/applications/com.palm.app.phoneprefs/app/models/SystemService.js
+++ /usr/palm/applications/com.palm.app.phoneprefs/app/models/SystemService.js
@@ -54,7 +54,7 @@
 }
 
 SystemService.getShortCutNumbers = function(callback) {
-	var keys = ['3DigitNumber', '4DigitNumber', '5DigitNumber', '6DigitNumber', '7DigitNumber'];
+	var keys = ['3DigitNumber', '4DigitNumber', '5DigitNumber', '6DigitNumber', '7DigitNumber', '8DigitNumber', '10DigitNumber'];
 	var request = new Mojo.Service.Request(SystemService.systemserviceIdentifier, {
 			method: 'getPreferences',
 			parameters: {'keys': keys}, 
